### ISSUE TITLE
linesGL with coordinateSystem:cartesian2d?

### ISSUE THREAD

**Issue by msbasanth** [ecomfe/echarts-gl#397]
Hi,

I am looking into nice example which updates canvas with huge number of lines progressively.
https://echarts.apache.org/examples/en/editor.html?c=linesGL-ny&gl=1

Good to see _appendData_ API, it matches one usecase which I am looking into. For an ECG monitor where I am getting **2000+ lines** to render per second.
![ECGWaveforms](https://user-images.githubusercontent.com/17777813/110758952-17f95c00-8273-11eb-97d8-501b0a1f2c77.gif)

Currently I can see "coordinateSystem: 'geo'", whether I can use "cartesian2d" or "cartesian3d" with "linesGL?

We wanted to give line a width, but when I updated lineStyle it is not changing line width. Instead I could see lines getting connected (line-strips) as shown below.

![image](https://user-images.githubusercontent.com/17777813/110759352-863e1e80-8273-11eb-82d9-8a2fd87f054f.png)

![image](https://user-images.githubusercontent.com/17777813/110759476-a66ddd80-8273-11eb-8e93-a7dcfc45e7a0.png)

This is how we updated the width,

```
lineStyle: {
            color: 'orange',
            opacity: 0.3,
            **width : 2**
        }
```

Whether we have option to add effect in linesGL? -  we need a a bloom effect (a glow-like effect).

Could you please point to documentation of Series type: _linesGL?_

Thanks
Basanth

**Comment by pissang** [comment]
I'm not sure your scenario. But I will suggest you using `lines` instead of `linesGL`

**Comment by msbasanth** [comment]
@pissang Thanks for your suggestion I will give a try with **lines**

**Comment by msbasanth** [comment]
@pissang 

Yes, I could see _lines_ series is closer to my usecase as it allows with few limitations observed. Really happy about the rendering performance and ability to append the objects into the view.

* I am not able to clear previously rendered objects 
* As the number of objects getting rendered increases, seeing a degradation in performance of waveform rendering
* For removing waveform section I had to render with background color which again adds to the number of objects in view.

Compared to lines linesGL  looks to be handling more number of objects, but with few missing options for using a realtime ECG rendering usecase,
* not able to set line width
* Should be able to remove objects inside view (basically from WebGL context)
